spec/models/manageiq/providers/openstack/cloud_manager/template_spec.rb

```diff
@@ -34,6 +35,22 @@
       end
     end
   end
+
+  context 'when update_image' do
+    before do
+      allow(ExtManagementSystem).to receive(:find).with(ems.id).and_return(ems)
+      allow(ems).to receive(:with_provider_connection).with(:service => 'Image').and_yield(service)
```

Comment:
@aufi everytime the `ems` here appears to be `nil` and tests don't pass. How can I get a right one? Maybe something goes wrong in line 2
```let(:ems) { FactoryGirl.create(:ems_openstack_with_authentication) }```
and FactoryGirl doesn't create right ems (some issues with authentication). Could you check it pls?